app/validation/modify_grow.py

```diff
@@ -0,0 +1,41 @@
+from typing import Any, Dict, List, Tuple, Optional
+from collections import Counter
+
+from app_config import AppConfig
+
+from app.models.grow import Grow
+from app.models.grow_phase import GrowPhase
+from app.models.recipe import Recipe
+from app.models.recipe_phase import RecipePhase
+
+
+def validate_modify_grow(app_config: AppConfig, message) -> Tuple[bool, str]:
+    if "grow_id" not in message:
+        return (False, "Grow ID not included")
```

Comment:
Consider using exceptions here

Reply:
I don't think we should throw an exception here since we're returning this error message to the client. Exception not needed, will just make error handling + returning correct error to client more difficult.